spec/services/course_groups/specialist_spec.rb

```diff
@@ -19,7 +19,9 @@
       end
     end
 
-    context "when date is between December of cohort start year and April of the next year" do
+    context "when date is between December of cohort start year and April of 2026" do
+      let(:cohort) { create(:cohort, start_year: 2025) }
```

Comment:
need the `travel_to` in a before block here?
(nice to have)

Reply:
I'd probably argue yes but without changing the whole file it will be inconsistent so for simplicity for now I'll leave it as is